src/core/cgroup.c

```diff
@@ -3751,9 +3751,7 @@ int manager_notify_cgroup_empty(Manager *m, const char *cgroup) {
 }
 
 int unit_get_memory_available(Unit *u, uint64_t *ret) {
```

Comment:
this changes behaviour for units of unexpected types or which aren't realized: instead of returning -ENODATA as before we'll just skip over them and return UINT64_MAX...

Reply:
Ah, right. The idea with `cgroup_path` is that an ancestor will have it realized (`-.slice` eventually). A missing `cgroup_context` is a bug -- fixed that.